modules/services/espanso.nix

```diff
@@ -1,19 +1,25 @@
 { pkgs, config, lib, ... }:
-
 let
-
   inherit (lib)
-    mkOption mkEnableOption mkIf maintainers literalExpression types platforms;
+    mkOption mkEnableOption mkIf maintainers literalExpression types platforms
+    mkRemovedOptionModule versionAtLeast;
 
   inherit (lib.hm.assertions) assertPlatform;
 
   cfg = config.services.espanso;
+  espansoVersion = cfg.package.version;
 
   yaml = pkgs.formats.yaml { };
-
 in {
-  meta.maintainers = with maintainers; [ lucasew ];
-
+  imports = [
+    (mkRemovedOptionModule [ "services" "espanso" "settings" ]
+      "Use services.espanso.configs and services.espanso.matches instead.")
+  ];
+  meta.maintainers = [
+    maintainers.lucasew
+    maintainers.bobvanderlinden
```

Comment:
Does @bobvanderlinden want to be added?

Reply:
Yes,  he said it was fine. https://discourse.nixos.org/t/how-to-override-service-espanso-module-for-home-manager/28551/12